src/serac/physics/tests/solid_statics_patch.cpp

```diff
@@ -187,7 +328,8 @@ double solution_error(const ExactSolution& exact_displacement, PatchBoundaryCond
 
   SolidMechanics<p, dim> solid(std::move(equation_solver), solid_mechanics::default_quasistatic_options, GeometricNonlinearities::On, "solid");
 
-  solid_mechanics::NeoHookean mat{.density=1.0, .K=1.0, .G=1.0};
+  //solid_mechanics::NeoHookean mat{.density=1.0, .K=1.0, .G=1.0};
```

Comment:
Is this where we were trying to get higher order patch tests working? If so, then this is an artifact of our experiments and should be reverted to the nonlinear model. Sam and I decided to postpone those tests for now, because we'll need to select higher precision quadrature rules to make it work.